recipe/meta.yaml

```diff
@@ -1,41 +1,51 @@
-{% set version = "1.900.31" %}
+{% set version = "2.0.14" %}
 
 package:
   name: jasper
   version: {{ version }}
 
 source:
-  url: https://github.com/mdadams/jasper/archive/version-1.900.31.tar.gz
-  sha256: 0099084dce9fa0a318b48a5ed2a12ac7cff3324f3fd4459833e6fe2ff583fb2c
+  url: http://www.ece.uvic.ca/~frodo/jasper/software/jasper-{{ version }}.tar.gz
+  sha256: 2a1f61e55afe8b4ce8115e1508c5d7cb314d56dfcc2dd323f90c072f88ccf57b
   patches:
-    # Skipping test due to https://github.com/mdadams/jasper/issues/179
-    # non linux64 is on a new enough system version of bash for this to not be an issue
-    - test.diff  # [linux64]
+    - CVE-2018-9055-330751ce.patch
 
 build:
   number: 0
   skip: True  # [win]
   run_exports:
     # no info.  Leaving at default behavior.
     - {{ pin_subpackage('jasper') }}
+  missing_dso_whitelist:                                         # [osx]
+    - /System/Library/Frameworks/GLUT.framework/Versions/A/GLUT  # [osx]
 
 requirements:
   build:
     - {{ compiler('c') }}
-    - automake             # [unix]
-    - autoconf             # [unix]
-    - libtool              # [unix]
-    - make                 # [unix]
+    - {{ cdt('mesa-libgl-devel') }}     # [linux]
+    - {{ cdt('mesa-dri-drivers') }}     # [linux]
+    - {{ cdt('libxdamage') }}           # [linux]
+    - {{ cdt('libselinux') }}           # [linux]
+    - {{ cdt('libxext') }}              # [linux]
+    - {{ cdt('libxfixes') }}            # [linux]
+    - {{ cdt('libxcb') }}               # [linux]
+    - {{ cdt('libxau') }}               # [linux]
+    - {{ cdt('libxi') }}                # [linux]
+    - {{ cdt('libxxf86vm') }}           # [linux]
```

Comment:
I guess all the CDTs need to be deselected too. `[not aarch64]` may help if ppc64le can find them

Reply:
I used `linux64` but let me test not `[not aarch64]`, hold on...